airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -56,7 +56,7 @@ def __init__(self,
             gcp_conn_id=bigquery_conn_id, delegate_to=delegate_to)
         self.use_legacy_sql = use_legacy_sql
         self.location = location
-        self.num_retries = self._get_field('num_retries', 5)
+        self.num_retries = self._get_field('num_retries', 5) or 5
```

Comment:
This code is repeated in many places. Does this not cause problems elsewhere?

Reply:
https://github.com/apache/airflow/search?q=self.num_retries&unscoped_q=self.num_retries

Reply:
You are right. I have moved it to GCP Base Hook now, what do you think?

Reply:
I'm fine with this, but we should move this to the GoogleDiscoveryApiHook class in the future.
https://github.com/apache/airflow/pull/5335/files

Reply:
👍